tests/php/Integration/Fixtures/Inline/mult.test

```diff
@@ -0,0 +1,6 @@
+--PHEL--
+(*)
+(* 1)
+(* 1 2)
+--PHP--
+(\Phel\Lang\Registry::getInstance()->getDefinition("phel\\core", "*"))();(1);(1 * 2);
```

Comment:
I think it would be interesting to add one more test with 3 arguments for those who have variadic/undefined number of arguments, because by default they are 1 or 2, so it would be helpful to see `(* 1 2 3) => (1 * 2 * 3)` too. The same applies to `+` and `-`, for example.
Maybe it doesn't worth it, and it's ok as it is... but I wanted to share this idea.

Reply:
I added these tests.